src/formInputs/DatePicker/MonthSelector/Month/index.tsx

```diff
@@ -10,36 +10,29 @@ export interface MonthSelectorMonthProps {
   readonly onClick: () => void;
 }
 
-const Component = (props: MonthSelectorMonthProps) => {
+export const MonthSelectorMonth = React.memo<MonthSelectorMonthProps>(props => {
   const { children, isSelected, onClick } = props;
   return (
     <Month isSelected={isSelected} onClick={onClick}>
       {children}
     </Month>
   );
-};
-
-export const MonthSelectorMonth = React.memo(Component);
+});
 
 const Month = styled.div<{ isSelected: boolean }>`
   display: flex;
   justify-content: center;
-  color: ${props => {
-    if (props.isSelected) {
-      return white;
-    }
-    return gray900;
-  }};
+  color: ${props => (props.isSelected ? white : gray900)};
   ${props =>
     props.isSelected
-      ? `
-    background-color: ${orange500}
-  `
-      : `
-
-  &:hover {
-    background-color: ${darken(0.1, white)};
-  }`}
+      ? css`
+          background-color: ${orange500};
+        `
+      : css`
+          &:hover {
+            background-color: ${darken(0.1, white)};
+          }
+        `};
```

Comment:
25번 줄도 이 안에서 같이 쓸 수 있지 않을까요?
```suggestion
      ? css`
          color: white;
          background-color: ${orange500};
        `
      : css`
          color: ${gray900};
          &:hover {
            background-color: ${darken(0.1, white)};
          }
        `};
```

Reply:
감사합니다!